Gandalan.IDAS.WebApi.Client/Settings/ConnectHub.cs

```diff
@@ -19,12 +19,13 @@ public async Task<HubResponse> GetEndpoints(string apiVersion = null, string env
                 (clientOs != null ? "clientOS=" + clientOs : "");
             try
             {
-                var response = await new HttpClient().GetStringAsync(requestUrl);
-                return JsonConvert.DeserializeObject<HubResponse>(response);
+                using var httpClient = new HttpClient();
+                var response = httpClient.GetStringAsync(requestUrl).GetAwaiter().GetResult();
```

Comment:
Can we switch to a real async method here - i.e. make it GetEndpointsAsync? I think that would be cleaner

Reply:
Done in 23a41b64c463d61eb2fb50a682576168c2136e68

Reply:
And in 4557792bc6ca0e85511174c9d72205f8de6c2b93

Reply:
Aaaand in 2e327ae450b92508da62bfd64d8ecb9518cac44d